bodhi/server/templates/update.html

```diff
@@ -376,7 +376,7 @@
         ${self.util.type2icon(update.type) | n}
       % endif
       % if update.critpath:
-      <span class="label label-danger"> <i class="fa fa-fire"></i></span>
+      <span data-toggle="tooltip" title="This update is in the Critical Path" class="label label-danger"> <i class="fa fa-fire"></i></span>
```

Comment:
Since "bugfix" will be lowercased on the other tooltip, should we lowercase "critical path"?